src/modules/flow/thingspeak/thingspeak.json

```diff
@@ -0,0 +1,149 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "thingspeak",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "BSD 3-Clause",
+    "version": "1"
+  },
+  "types": [
+    {
+      "category": "iot/cloud",
+      "description": "Execute the next Thingspeak Talkback command",
+      "methods": {
+        "open": "thingspeak_execute_open",
+        "close": "thingspeak_execute_close"
+      },
+      "name": "thingspeak/talkback-execute",
+      "options": {
+        "members": [
+          {
+            "data_type": "string",
+            "description": "Thingpeak API key",
+            "name": "api_key"
+          },
+          {
+            "data_type": "string",
+            "description": "Talkback ID",
+            "name": "talkback_id"
+          },
+          {
+            "data_type": "string",
+            "default": "https://api.thingspeak.com/",
+            "description": "Thingspeak endpoint. Defaults to public server",
+            "name": "endpoint"
+          },
+          {
+            "data_type": "int",
+            "default": 1,
+            "description": "Poll interval in seconds. Default to 1 request/second.",
```

Comment:
I'll change that. However, to keep the number of requests/second sane, the minimum value will be 1000ms.